### PR TITLE
feat: switch to generic secrets webhook

### PR DESCRIPTION
**Pull request by csatib02**
## Overview

- This PR changes `Vault-env` and `vault-secrets-webhook`references to secret-init and secrets-webhook references.

### CODE REVIEW

File: ADOPTERS.md
Comment (akijakya) [inline ADOPTERS.md:1]: I think we should leave this as is, as they probably still using the old webhook at this point 😊

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Bank-Vaults is an umbrella project which provides various tools for Cloud Native
 
 - Bank-Vaults CLI to make configuring Hashicorp Vault easier
 - [Vault operator](https://github.com/bank-vaults/vault-operator/) to make operating Hashicorp Vault on top of Kubernetes easier
```
Comment (akijakya) [inline README.md:25]:
```suggestion
- [Vault Operator](https://github.com/bank-vaults/vault-operator/) to make operating Hashicorp Vault on top of Kubernetes easier
```

File: README.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ Bank-Vaults is an umbrella project which provides various tools for Cloud Native
...
-- [Vault secrets webhook](https://github.com/bank-vaults/vault-secrets-webhook) to inject secrets directly into Kubernetes pods
+- [Secrets webhook](https://github.com/bank-vaults/secrets-webhook) to inject secrets directly into Kubernetes pods
```
Comment (akijakya) [inline README.md:26]:
```suggestion
- [Secrets Webhook](https://github.com/bank-vaults/secrets-webhook) to inject secrets directly into Kubernetes pods
```

**Review by akijakya — CHANGES_REQUESTED** [review]
Great work so far! 💪

**Review by ramizpolic — DISMISSED** [review]

File: Makefile [hunk 3]
```diff
@@ -16,7 +16,7 @@ PACKAGE = github.com/bank-vaults/bank-vaults
 BINARY_NAME ?= bank-vaults
 DOCKER_REGISTRY ?= ghcr.io/bank-vaults
 DOCKER_IMAGE = ${DOCKER_REGISTRY}/bank-vaults
-WEBHOOK_DOCKER_IMAGE = ${DOCKER_REGISTRY}/vault-secrets-webhook
```
Comment (ramizpolic) [inline Makefile:19]: this var is migration split leftover, can be removed

**Review by ramizpolic — APPROVED** [review]

**Review by akijakya — APPROVED** [review]

**Pull request merged**